runtime/server/server.go

```diff
@@ -241,3 +246,49 @@ func (s *Server) Ping(ctx context.Context, req *runtimev1.PingRequest) (*runtime
 	}
 	return resp, nil
 }
+
+func InitOpenTelemetry() error {
```

Comment:
Should this not integrate with the server lifecycle? E.g. how will this work in testing when there are multiple servers? How will the exporters be closed/flushed correctly when the process stops?